spec.md

```diff
@@ -379,26 +388,26 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id -d
 | Status Code  | Description  |
 |---|---|
 | 201 Created  | Service instance has been provisioned. The expected response body is below.  |
-| 200 OK  |  May be returned if the service instance already exists and the requested parameters are identical to the existing service instance. The expected response body is below. |
+| 200 OK  |  MAY be returned if the service instance already exists and the requested parameters are identical to the existing service instance. The expected response body is below. |
```

Comment:
+1 on tightening up the language, my reading is that a 202 should be returned to the async request if one is already ongoing.